lib/controller.js

```diff
@@ -355,7 +355,8 @@ controller.provisionTessel = function(opts) {
             }
           });
         } else {
-          reject('Keys already exist. Refusing to overwrite them.');
+          // ~/.tessel exists with keys
+          resolve();
```

Comment:
Yes, I'm in the midst of adding it and fixing the other tests but it's been more involved than I expected.